src/controllers/categoryOfMeasuring.js

```diff
@@ -0,0 +1,11 @@
+const createCategoriesOfMeasuringsModel = require("../models/categoriesofmeasurng.js");
+
+const { DataTypes } = require("sequelize");
+const { db } = require("../databaseSetup");
+
+const CategoriesOfMeasurings = createCategoriesOfMeasuringsModel(db, DataTypes);
+
+const getAllCategoriesOfMeasurings = async (req, res) =>
+  res.json(await CategoriesOfMeasurings.findAll());
+
+module.exports = getAllCategoriesOfMeasurings;
```

Comment:
Please replace it with non-default export, because this controller can have more that one handler

Reply:
Replaced ✔️